docs/content/en/docs/concepts/architecture/keptn-apps/_index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: KeptnApp and KeptnWorkload resources
+linktitle: Keptn Applications and Keptn Workloads
+description: How Keptn applications work
+weight: 50
+cascade:
+---
+
+## Keptn Workloads
+
+A
+[KeptnWorkload](../../../crd-ref/lifecycle/v1alpha3/#keptnworkload)
+resource augments a Kubernetes
+[Workload](https://kubernetes.io/docs/concepts/workloads/)
+with the ability to handle extra phases.
+It can execute the pre- and post-deployment evaluations of a Workload
+and run pre- and post-deployment tasks.
+
+In its state, it tracks the currently active `Workload Instances`,
+(`Pod`, `DaemonSet`, `StatefulSet`, and `ReplicaSet` resources),
+as well as the overall state of the Pre Deployment phase,
+which the scheduler can use to determine whether the pods belonging to a workload should proceed.
+When it detects that the referenced object has reached its desired state
+(e.g. all pods of a deployment are up and running),
+it knows that a`PostDeploymentCheck` can be triggered.
+
+The KeptnWorkload resources are created automatically
+by the mutating webhook as soon as a pod for the workload
```

Comment:
```suggestion
by the mutating webhook as soon as you apply their manifest
```

Reply:
Oh, now I see -- too many comments for the same paragraph ;-)  Check what I have now -- I went for the passive "as soon as the workload manifest is applied" -- I'm thinking that the user doesn't always directly apply a manifest...